src/main/resources/migration/changelog-v5.6.0.xml

```diff
@@ -107,4 +107,202 @@
+            <column autoIncrement="true" name="ID" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="PROTOCOL_PROPERTIES_PK"/>
+            </column>
+            <column name="TYPE" type="VARCHAR(16)"/>
+            <column name="VERSION" type="VARCHAR(16)"/>
+        </createTable>
+
+        <createTable tableName="CRYPTO_FUNCTIONS">
+            <column autoIncrement="true" name="ID" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="CRYPTO_FUNCTIONS_PK"/>
+            </column>
+            <column name="ALGORITHM_PROPERTY_ID" type="BIGINT"/>
+            <column name="CRYPTO_FUNCTION" type="VARCHAR(32)"/>
+        </createTable>
+
+        <createTable tableName="PROTOCOL_CIPHER_SUITES">
+            <column name="PROTOCOL_PROPERTY_ID" type="BIGINT"/>
+            <column name="CIPHER_SUITE_ID" type="BIGINT"/>
+        </createTable>
+
+        <createTable tableName="CIPHER_SUITE">
+            <column autoIncrement="true" name="ID" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="CIPHER_SUITE_PK"/>
+            </column>
+            <column name="NAME" type="VARCHAR(64)"/>
+        </createTable>
+
+        <createTable tableName="CIPHER_SUITE_ALGORITHM">
+            <column name="CIPHER_SUITE_ID" type="BIGINT"/>
+            <column name="ALGORITHM" type="VARCHAR(64)"/>
+        </createTable>
+
+        <createTable tableName="CIPHER_SUITE_IDENTIFIER">
+            <column name="CIPHER_SUITE_ID" type="BIGINT"/>
+            <column name="IDENTIFIER" type="VARCHAR(8)"/>
+        </createTable>
+
+        <createTable tableName="PROTOCOL_IKEV2_TYPES">
+            <column name="PROTOCOL_PROPERTY_ID" type="BIGINT"/>
+            <column name="IKEV2_TYPE_ID" type="BIGINT"/>
+        </createTable>
+
+        <createTable tableName="IKEV2_TYPE">
+            <column autoIncrement="true" name="ID" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="IKEV2_TYPE_PK"/>
+            </column>
+            <column name="TYPE" type="VARCHAR(16)"/>
+        </createTable>
+
+        <createTable tableName="IKEV2_REF">
+            <column name="IKEV2_TYPE_ID" type="BIGINT"/>
+            <column name="REF" type="VARCHAR(64)"/>
+        </createTable>
+
+        <createTable tableName="CRYPTO_REFS">
+            <column autoIncrement="true" name="ID" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="CRYPTO_REF_PK"/>
+            </column>
+            <column name="PROTOCOL_PROPERTY_ID" type="BIGINT"/>
+            <column name="CRYPTO_REF" type="VARCHAR(64)"/>
+        </createTable>
+
+        <createTable tableName="COMPONENT_OCCURRENCES">
+            <column name="COMPONENT_ID" type="BIGINT"/>
+            <column name="OCCURRENCE_ID" type="BIGINT"/>
+        </createTable>
```

Comment:
The PR currently adds crypto-related data fields that our scanner (https://github.com/IBM/cbomkit) generates. We can drop OCCURRENCES because the location of crypto assets in source is probably not important in DT.